test/reporters/json-stream.spec.js

```diff
@@ -3,27 +3,46 @@
 var reporters = require('../../').reporters;
 var JSONStream = reporters.JSONStream;
 
+var runnerEvent = require('./helpers').runnerEvent;
+var makeExpectedTest = require('./helpers').makeExpectedTest;
+
 describe('Json Stream reporter', function () {
   var runner;
   var stdout;
   var stdoutWrite;
 
+  var expectedTitle;
+  var expectedFullTitle;
+  var expectedDuration;
+  var currentRetry;
+  var expectedTest;
+  var expectedErrorMessage;
+  var expectedErrorStack;
+  var expectedError;
+
   beforeEach(function () {
     stdout = [];
     runner = {};
     stdoutWrite = process.stdout.write;
     process.stdout.write = function (string) {
       stdout.push(string);
     };
+
+    expectedTitle = 'some title';
```

Comment:
It doesn't make a lot of sense to declare all this above then define it here, does it?  Can we not just define it all above?